static/app/views/alerts/rules/issue/details/issuesList.tsx

```diff
@@ -23,134 +25,103 @@ type GroupHistory = {
   lastTriggered: string;
 };
 
-type Props = DeprecatedAsyncComponent['props'] &
-  DateTimeObject & {
-    organization: Organization;
-    project: Project;
-    rule: IssueAlertRule;
-    cursor?: string;
-  };
-
-type State = DeprecatedAsyncComponent['state'] & {
-  groupHistory: GroupHistory[] | null;
+type Props = DateTimeObject & {
+  organization: Organization;
```

Comment:
should we remove the `organization` prop since we call `useOrganization()` below anyway?